proto/hummock.proto

```diff
@@ -55,6 +55,7 @@ message IntraLevelDelta {
   uint64 l0_sub_level_id = 2;
   repeated uint64 removed_table_ids = 3;
   repeated SstableInfo inserted_table_infos = 4;
+  repeated uint64 removed_table_object_ids = 5;
```

Comment:
Is it possible to calculate this value when we want to use it by comparing the diff of two hummock version instead of storing in a field here?

Reply:
Actually it can be calculated in a `HummockVersionDelta` if we assume the `InsertLevelDelta` will always be the last entry in the `GroupDeltas`.

https://github.com/risingwavelabs/risingwave/pull/10766/files#diff-146e602f03acfd5f0d45bb27201afd6e7a62ebd99a7d1af5bc2f2f660b36c7e3R2728-R2737

Reply:
Can we assume it?

Reply:
Plus, we cannot tell the difference between trivial move and compaction with delta info.